docs/self-hosted-apps/reverseproxy.md

```diff
@@ -0,0 +1,29 @@
+# Self Hosted Reverse Proxy
+
+As always, check out [awesome-selfhosted](https://github.com/awesome-selfhosted/awesome-selfhosted) but here are our favourites.
+
+## Traefik
+
+**Website:** [https://traefik.io](https://traefik.io)
+
+**Project Page:** [containous/traefik](https://github.com/containous/traefik)
+
+**Docker Hub:** [traefik](https://hub.docker.com/_/traefik)
+
+### Reviews and Related Links
+   * 
+   *
+
+
+## Linuxserver.io's Nginx Let's Encrypt Reverse Proxy
```

Comment:
This section should really be about Nginx itself, and then list the LSIO container underneath, along with why you might want to use it